packages/babel-generator/test/fixtures/flow/call-properties/expected.js

```diff
@@ -1,5 +1,14 @@
-var a: { (): number };
-var a: { (): number };
-var a: { y: string, (): number, (x: string): string, };
-var a: { <T>(x: T): number };
-interface A { (): number };
+var a: { (): number
```

Comment:
Weird asymmetry on these objects. I think if the closing bracket is on its own line, the opening one should be too.